static/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -120,6 +123,16 @@ class ColumnEditCollection extends React.Component<Props, State> {
 
   handleUpdateColumn = (index: number, column: Column) => {
     const newColumns = [...this.props.columns];
+    if (column.kind === 'equation') {
+      this.setState(prevState => {
+        const error = prevState.error;
+        error[index] = parseArithmetic(column.field).error;
+        return {
+          ...prevState,
+          error,
+        };
+      });
+    }
```

Comment:
Does `error` need to be cleared/reset as new columns are added, removed, reordered, etc.?